### DISCUSSION TITLE
takingcredit

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#14] [Announcements]
# takingcredit

Exploring the paradox of forming ideas and giving credit, focusing on the concept of Constructivism.

https://nonzerosum.games/takingcredit.html?fbclid=IwAR2VinUwxE_vJq35isOzZFLBSuSC42owvusxZHeOvlRdas0OROMato4jJ9E

**Comment by andrewtaneglen** [comment]
This is really interesting . Another angle to it is what part is the most enjoyable part of some coming up with a good idea: 1) The moment you think of it, or 2) The moment you mention it to someone capable of critiquing the idea, and they agree it is a good idea. Keeping it to yourself protects you from the risk of the idea being shown as wrong (or already had by someone else). Another use for the 'to be or not to be' analogy I guess.
Another scenario would be when you've done 99% of the research work, and then someone else takes the next logic step, coming up with the more exciting/famous outcome. Newton's 'standing on the shoulders of giants' line is perhaps the most appropriate attitude to have in these contexts.

**Reply by jamesstephenbrown** [reply]
I really think the most rewarding part is seeing someone get your idea, it's nice if they agree, but I think just feeling you got an idea across is sort of what our brains (I can only speak for mine) crave. Everyone wants to be understood I guess, and communicating an idea that you feel is unique and having that be understood  has got to be the best expression of "being understood" in general. But who knows, it's one of those "what are we doing all this for again?" questions.

**Reply by jamesstephenbrown** [reply]
Standing on the shoulders of giants = progress. Standing on the shoulders of giant = bastard (from the giant's point of view).
